I use your latest commit to install `cppslippi:x64-windows` failed with the following error:
```
-- Configuring x64-windows
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: "C:/Program Files/CMake/bin/cmake.exe" --build . --config Debug --target install -- -v -j9
    Working Directory: E:/Lily/vcpkg/buildtrees/cppslippi/x64-windows-dbg
    See logs for more information:
      E:\Lily\vcpkg\buildtrees\cppslippi\install-x64-windows-dbg-out.log
      E:\Lily\vcpkg\buildtrees\cppslippi\install-x64-windows-dbg-err.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/cppslippi/portfile.cmake:15 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)
```
**install-x64-windows-dbg-err.log**
```
ninja: error: manifest 'build.ninja' still dirty after 100 tries, perhaps system time is not set
```

--- a/ports/cppslippi/portfile.cmake
+++ b/ports/cppslippi/portfile.cmake
@@ -1,0 +1,21 @@
+vcpkg_from_sourceforge(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO            cppslippi
+    FILENAME        "CppSlippi-${VERSION}.zip"
+    SHA512          5057758ed7da1f8d3bbdcd0eb783e93aa07d501a5333c48707f588bd3370551fce65a22074da9cbe4aa69fa32fa6c826a2aa039911cff7be5b8548842d135ece
+    NO_REMOVE_ONE_LEVEL
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH ${SOURCE_PATH}
+    OPTIONS
+        -DBUILD_TESTING=False
+)
+
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/CppSlippi)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.txt")
+
+file(COPY "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+file(COPY "${SOURCE_PATH}/README.md" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")

--- a/ports/cppslippi/usage
+++ b/ports/cppslippi/usage
@@ -1,0 +1,4 @@
+The package cppslippi provides CMake targets:
+
+    find_package(CppSlippi CONFIG REQUIRED)
+    target_link_libraries(main PRIVATE CppSlippi::CppSlippi)

--- a/ports/cppslippi/vcpkg.json
+++ b/ports/cppslippi/vcpkg.json
@@ -1,0 +1,18 @@
+{
+  "name": "cppslippi",
+  "version": "1.0.3.14",
+  "description": "C++ Slippi replay file parser.",
+  "homepage": "https://sourceforge.net/projects/cppslippi/",
+  "license": "MIT",
+  "dependencies": [
+    "nlohmann-json",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ]
+}

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -165,6 +165,9 @@ cppmicroservices:arm-uwp=fail
 cppmicroservices:x64-uwp=fail
 cpp-netlib:arm-uwp=fail
 cpp-netlib:x64-uwp=fail
+# Requires full C++20 support, currently absent from CI for these triplets.
+cppslippi:x64-linux=fail
+cppslippi:x64-osx=fail
 cppcoro:x64-linux=fail
 cppcoro:arm-uwp=fail
 cppcoro:x64-uwp=fail

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1776,6 +1776,10 @@
       "baseline": "2.10.18",
       "port-version": 2
     },
+    "cppslippi": {
+      "baseline": "1.0.3.14",
+      "port-version": 0
+    },
     "cpptoml": {
       "baseline": "v0.1.1",
       "port-version": 2

--- a/versions/c-/cppslippi.json
+++ b/versions/c-/cppslippi.json
@@ -1,0 +1,9 @@
+{
+  "versions": [
+    {
+      "git-tree": "3cb6381580bcf82551009114b87021b0b1bc9885",
+      "version": "1.0.3.14",
+      "port-version": 0
+    }
+  ]
+}

